cmd/oci-runtime-tool/validate.go

```diff
@@ -27,8 +36,13 @@ var bundleValidateCommand = cli.Command{
 		}
 
 		if err := v.CheckAll(); err != nil {
+			removed, err := specerror.FilterError(err, complianceLevel)
```

Comment:
I think `removed` is not so easy to understand, how about `filtered`?

Reply:
> I think `removed` is not so easy to understand, how about `filtered`?

I'm open to rewording, but `FilterError` is [currently][1] using `removed` for “these were removed from the input error due to their low level” and `filtered` for “the high-level and non-`specerror` stuff you still need to worry about”.  If we decide to use `filtered` for the low-level stuff, what should we use for the high-level and non-`specerror` stuff?

[1]: https://github.com/opencontainers/runtime-tools/pull/492/files#diff-87c0b1eb29ae71890bc6a05f207759edR175

Reply:
How about just printing `removed errors` in FilterError, FilterError just returns filtered errors?

Reply:
> How about just printing `removed errors` in FilterError, FilterError just returns filtered errors?

I'd rather not make UI decisions in a library function.  Maybe the caller wants to silently ignore low-level errors.  Or maybe they want to present them as TAP diagnostics (#439).  Returning the low-level error slice leaves that sort of thing up to the caller.

Reply:
but `removed, err := specerror.FilterError(err, complianceLevel)` looks very strange. That looks like `removed` is the expected  result, err represents error message.

Reply:
> but `removed, err := specerror.FilterError(err, complianceLevel)` looks very strange. That looks like removed is the expected result, err represents error message.

So maybe `low, high, alwaysNil := specerror.FilterError(...)`?

Reply:
I think `alwaysNil` is also not so good. How about just returning a struct includes `filtered` and `low` errors?

Reply:
> I think `alwaysNil` is also not so good. How about just returning a struct includes `filtered` and `low` errors?

Works for me; I'll reroll in a few hours.

Reply:
I've rerolled to return a helper struct in 1cbca20 → 6089f63.